### PR TITLE
Docs for Collection#at behavior

### PR DESCRIPTION
**Pull request by FishbaitHarry**
A short fix for https://github.com/jashkenas/backbone/issues/3789 - I hope not to take unnecessarily too much space.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2048,7 +2048,8 @@ <h2 id="Collection">Backbone.Collection</h2>
       <br />
       Get a model from a collection, specified by index. Useful if your collection
       is sorted, and if your collection isn't sorted, <b>at</b> will still
-      retrieve models in insertion order.
+      retrieve models in insertion order. When passed a negative index, it
+      will retrieve model from the back of collection.
```
Comment (jridgewell) [inline index.html]: "the model from the back of the collection" please.

**Comment by jridgewell** [comment]
Thanks for the doc PR, Just have that one grammar change.

**Comment by jsatk** [comment]
+1

**Comment by FishbaitHarry** [comment]
Ok, I updated my branch.

**Comment by jridgewell** [comment]
Thanks!

**Pull request merged**